internal/store/files/recog_place_map.go

```diff
@@ -52,23 +64,28 @@ func RecogPlaceMap() (map[string]*pb.RecogPlaces, error) {
 				"Wrong RecogPlaces CSV record: %v", record)
 		}
 
-		recogPlace := &pb.RecogPlace{
-			ContainingPlaces: strings.Split(strings.ReplaceAll(record[3], " ", ""), ","),
-		}
-
 		// DCID.
-		recogPlace.Dcid = strings.TrimSpace(record[0])
-		if recogPlace.Dcid == "" {
+		dcid := strings.TrimSpace(record[0])
+		if dcid == "" {
 			return nil, status.Errorf(codes.FailedPrecondition,
 				"Empty DCID for CSV record: %v", record)
 		}
+		recogPlace := &pb.RecogPlace{Dcid: dcid}
+
+		// Containing places.
+		containingPlaces := strings.Split(strings.ReplaceAll(record[3], " ", ""), ",")
```

Comment:
For the row: `<dcid>,Chennai, District, "Tamil Nadu, India",6000000`, will "Tamil Nadu" be addd as "TamilNadu"?  If so, will that match the query string?

Reply:
Note that this line of code deals with "containingPlaces", which is a list of DCIDs (e.g. "country/IND,wikidataId/XXX,", etc." - Your comment is about names (which contains spaces).

Reply:
Ah, got it.